packages/contentful--app-scripts/src/upload/create-app-bundle.ts

```diff
@@ -27,7 +27,16 @@ export async function createAppBundleFromUpload(settings: UploadSettings, appUpl
       functions,
     });
   } catch (err: any) {
-    showCreationError('app upload', err.message);
+    try {
+      const message = JSON.parse(err.message);
+      if (message['status'] == 403) {
```

Comment:
wondering if it might be worth doing something like 
```suggestion
      if (message['status'] == 403 && message.details?.reasons) {
```

Reply:
to confirm the reasons are there before attempting to use them - just in case a weird 403 comes through that isn't shaped the right way

Reply:
Good catch, thank you!